index.html

```diff
@@ -64,71 +64,195 @@
 				padding:15px 0;
 				clear:both;
 			}
+
+			#builds table {
+				width:100%;
+				font-size: small;
+			}
+
+			#builds tr.even
+			{
+				background-color:#efefef;
+			}
+
+			#builds td {
+				vertical-align: top;
+				padding:0 10px 0 0;
+			}
+
+			#builds td.date, #builds td.commit {
+					white-space: nowrap;
+					width:1%;
+			}
+			#builds th {
+				text-align: left;
+			}
+
+			#builds th.branch
+			{
+				font-size: large;
+			}
 		</style>
 		<script type="text/javascript">
 			$( document ).ready(function() {
+				function keys(obj)
+				{
+						var keys = [];
+						for(var key in obj)
+						{
+								if(obj.hasOwnProperty(key))
+								{
+										keys.push(key);
+								}
+						}
+						return keys;
+				}
+
+				function format_size(bytes) {
+					var i = 0;
+					var units = ['B', 'kB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
```

Comment:
All non-apple users use KiB (though, it's written as KB). Afaik apple
started using 1000 based units in 2006 or so.

On Wed, Aug 26, 2015 at 7:28 PM, Jan Holthuis <notifications@github.com>
wrote:

> In index.html
> <https://github.com/reicast/reicast-builds/pull/3#discussion_r38009880>:
>
> > +				function keys(obj)
> > +				{
> > +						var keys = [];
> > +						for(var key in obj)
> > +						{
> > +								if(obj.hasOwnProperty(key))
> > +								{
> > +										keys.push(key);
> > +								}
> > +						}
> > +						return keys;
> > +				}
> > +
> > +				function format_size(bytes) {
> > +					var i = 0;
> > +					var units = ['B', 'kB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
>
> We could, but I thought that KB is more known than KiB.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/reicast/reicast-builds/pull/3/files#r38009880>.
>



-- 
~skmp